calculator.js

```diff
@@ -1,28 +1,61 @@
 
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
-}
-// '3 + 5 = 8'
+const calculateAdd = function(input) {
+  let result = input.num1 + input.num2
+  console.log(`${input.num1} + ${input.num2} = ${result}`);
+   return;
+};
+const calculateSub = function(input ){
+  let result = input.num1 - input.num2
+  return console.log(`${input.num1} - ${input.num2} = ${result}`);
+  return;
+};
+const calculateDiv = function(input) {
+  if (input.num1 === 0 || input.num2 === 0){
+  console.log('You cannot divide by zero');
+   return;
+  }
+  let result = input.num1 / input.num2
+  console.log(`${input.num1} / ${input.num2} = ${result}`);
+   return;
+};
+const calculateMult = function(input) {
+  let result = input.num1 * input.num2
+  console.log(`${input.num1} * ${input.num2} = ${result}`);
+  return;
+  };
 
-const calculateUserInput = function (error, promptInput) {
-  console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
-
-  // Questions to ask and answer:
-  // What is promptInput?
-  // What data type? What does it hold? What does it represent?
-  // How do we read values from it? What syntax?
-  // How can we use it?
-  // Can we call our existing functions now, inside of this function?
-}
-
-// Example manual testing of calculator.  
-calculateUserInput({}, {
-  num1: 3,
-  num2: 4,
-  operation: 'add',
-});
+const notANumber = function(input) {
+	const validatedInput = {operation: input.operation}
+  validatedInput.num1 = parseInt(input.num1)
+  validatedInput.num2 = parseInt(input.num2)
```

Comment:
The semi-colons you have are all placed correctly. In addition, semi-colons should follow assignment statements.

```suggestion
  validatedInput.num1 = parseInt(input.num1);
  validatedInput.num2 = parseInt(input.num2);
```